project/Dependencies.scala

```diff
@@ -0,0 +1,359 @@
+import sbt._
+import sbt.Keys._
+
+object Dependencies {
+
+  // Version numbers
+  val PlayVersion = "2.5.10"
+  val AkkaVersion = "2.4.17"
+  val AkkaPersistenceCassandraVersion = "0.22"
+  val ScalaTestVersion = "3.0.1"
+  val JacksonVersion = "2.7.8"
+  val CassandraAllVersion = "3.0.9"
+  val GuavaVersion = "19.0"
+  val MavenVersion = "3.3.9"
+  val NettyVersion = "4.0.40.Final"
+  val KafkaVersion = "0.10.0.1"
+  val AkkaStreamKafka = "0.13"
+  val Log4j = "1.2.17"
+  val ScalaJava8CompatVersion = "0.7.0"
+
+  // Specific libraries that get reused
+  val scalaTest = "org.scalatest" %% "scalatest" % ScalaTestVersion
+  val guava = "com.google.guava" % "guava" % GuavaVersion
+  val log4J = "log4j" % "log4j" % Log4j
+  val scalaJava8Compat = "org.scala-lang.modules" %% "scala-java8-compat" % ScalaJava8CompatVersion
+
+
+  // Dependencies for each module
+  val api = libraryDependencies ++= Seq(
+    "org.scala-lang.modules" %% "scala-parser-combinators" % "1.0.4",
+    "com.typesafe.akka" %% "akka-actor" % AkkaVersion,
+    "com.typesafe.akka" %% "akka-slf4j" % AkkaVersion,
+    "com.typesafe.akka" %% "akka-stream" % AkkaVersion,
+    "com.typesafe.play" %% "play" % PlayVersion
+  )
+
+  val `api-javadsl` = libraryDependencies ++= Seq(
+    "com.typesafe.play" %% "play-java" % PlayVersion,
+    // todo Remove as part of #530
+    guava,
+    "org.pcollections" % "pcollections" % "2.1.2",
+    scalaTest % Test,
+    "com.fasterxml.jackson.module" % "jackson-module-parameter-names" % JacksonVersion % Test
+  )
+
+  val `api-scaladsl` = libraryDependencies ++= Seq(
+    scalaTest % Test
+  )
+
+  val immutables = libraryDependencies += "org.immutables" % "value" % "2.3.2"
+
+  val jackson = libraryDependencies ++= Seq(
+    "com.fasterxml.jackson.module" % "jackson-module-parameter-names" % JacksonVersion,
+    // todo Many of these won't be needed when #530 is done
+    "com.fasterxml.jackson.core" % "jackson-core" % JacksonVersion,
+    "com.fasterxml.jackson.core" % "jackson-annotations" % JacksonVersion,
+    "com.fasterxml.jackson.core" % "jackson-databind" % JacksonVersion,
+    "com.fasterxml.jackson.datatype" % "jackson-datatype-pcollections" % JacksonVersion,
+    "com.fasterxml.jackson.datatype" % "jackson-datatype-guava" % JacksonVersion,
+    "com.fasterxml.jackson.datatype" % "jackson-datatype-jdk8" % JacksonVersion,
+    "com.fasterxml.jackson.datatype" % "jackson-datatype-jsr310" % JacksonVersion,
+    "com.typesafe.akka" %% "akka-testkit" % AkkaVersion % "test",
+    scalaTest % Test,
+    "com.novocode" % "junit-interface" % "0.11" % "test"
```

Comment:
Should `akka-testkit`, `scalaTest` and `junit-interface` be part of `val jackson' ?